package.json

```diff
@@ -24,6 +27,7 @@
     "file-loader": "^2.0.0",
     "html-loader": "^0.5.5",
     "html-webpack-plugin": "^3.2.0",
+    "nodemon": "^1.18.6",
```

Comment:
nodemon should be used for development purpose only and should be installed globally